protocols/jabber/jabber.c

```diff
@@ -83,6 +81,12 @@ static void jabber_init(account_t *acc)
 	s = set_add(&acc->set, "server", NULL, set_eval_account, acc);
 	s->flags |= SET_NOSAVE | ACC_SET_OFFLINE_ONLY | SET_NULL_OK;
 
+	if (strcmp(acc->prpl->name, "hipchat") == 0) {
+		set_setstr(&acc->set, "server", "chat.hipchat.com");
```

Comment:
Because of @binwiederhier's bug, fixed in a51220df24aa87b638ef0064e3078ae8da644b4f, which says "so set_eval_account gets called" in the commit log.

Turns out it *looked* like it was set to the right value when it was a default, but without set_eval_account, it kept trying to do a SRV (in my case, it tried to connect to a gtalk server, heh).

It only applied to newly created accounts, too, so wouldn't have noticed otherwise. Sneaky bug.